src/components/Property/Property.js

```diff
@@ -41,12 +52,13 @@ export default class Property extends PureComponent {
       classes
     } = this.props
 
-    let {name, enumValues} = this.props
+    const name = this.state.name
+    let enumValues
 
-    // If enumValues only has one single value, append the single value to name, and not display enum values
-    if (enumValues && enumValues.length === 1) {
-      name = `${name} = "${enumValues[0]}"`
-      enumValues = []
+    if (this.state.isEnumTrimmed) {
```

Comment:
This prop is accessed twice, did you want to destructure `this.state` to get it?